packages/maskbook/src/plugins/ITO/hooks/usePoolPayload.ts

```diff
@@ -3,13 +3,13 @@ import { PluginITO_RPC } from '../messages'
 import { useValueRef } from '../../../utils/hooks/useValueRef'
 import type { JSON_PayloadInMask } from '../types'
 
-let payloadRef = new ValueRef<JSON_PayloadInMask | null>(null)
+let storage = new Map<string, ValueRef<JSON_PayloadInMask>>()
```

Comment:
Here we don't need a cache, retry the pool if needed, so just store each pool by its own pid.